aea/cli/check_packages.py

```diff
@@ -226,14 +207,88 @@ def check_description(configuration_file: Path) -> None:
         raise EmptyPackageDescription(configuration_file)
 
 
-if __name__ == "__main__":
-    all_packages_ids_ = find_all_packages_ids()
+def check_handlers(config_file: Path, handler_config: Any) -> None:
```

Comment:
This doesn't work here. It is specific to our other repo. Please remove here and add there.